ansible/roles/openstack/init_vm/vars/CentOS-7.yml

```diff
@@ -0,0 +1,5 @@
+openstack_packages:
+  - NetworkManager
+  - centos-release-ansible-29
+  - python-rhsm-certificates
```

Comment:
Do we still need such a package ?

Reply:
I didn't test the installation without them. They were there before that's why I left them.

Reply:
This package "python-rhsm-certificates" was installed to get the red hat root certificate needed to install RHEL or EPEL repo if I remember. Otherwise, an error was reported during yum package installation. I suppose that RHEL subscription is better integrated and package not needed anymore. To be tested

Reply:
ok, I'll test without it.

Reply:
removed all the packages and the installation was successfull.